modules/ROOT/pages/install-manual.adoc

```diff
@@ -207,6 +207,7 @@ The following variables can be defined based upon your organization's requiremen
 {sp} +
 {sp} +
 If the host operating system is RHEL 8, do not set to `false` or installation will fail. | `true`
+| `RTF_Private_Interface` | Allows private IP address to be discovered from the network interface. | `eth0`
```

Comment:
This needs to be all caps. RTF_PRIVATE_INTERFACE

Reply:
Fixed, thanks